docs/reference-guide/index.md

```diff
@@ -0,0 +1,22 @@
+---
+myst:
+  html_meta:
+    "description": "Plone reference guide"
+    "property=og:description": "Plone reference guide"
+    "property=og:title": "Plone reference guide"
+    "keywords": "Plone, reference, guide"
+---
+
+# Reference guide
+
+This reference guide details application programming interfaces (APIs), functions, modules, and objects included in Plone, describing what they are and what they do.
+Its audience is primarily developers of Plone and its projects and add-ons.
```

Comment:
@stevepiercy I'm not sure this structure is right. Just because a section of docs covers an API doesn't mean that it's all reference. The lists of endpoints / functions are reference but there's also conceptual and how-to material in there.

Reply:
There is a way to solve this by ignoring `plone.restapi`'s and `plone.api`'s `index.md` files and using custom ones in `plone/documentation`. It's hard to describe without an example, but it works.